### PR TITLE
Feat Mobile: display busy in hashtag channel voice

### PR DESCRIPTION
**Pull request by vyhuynhngoc**

### CODE REVIEW

File: apps/mobile/src/app/screens/home/homedrawer/components/ChannelList/ChannelListItem/index.tsx [hunk 1]
```diff
@@ -115,6 +122,12 @@ export const ChannelListItem = React.memo((props: IChannelListItemProps) => {
 					<Text style={[styles.channelListItemTitle, isUnRead && styles.channelListItemTitleActive]} numberOfLines={1}>
 						{props.data.channel_label}
 					</Text>
+
+					{checkVoiceStatus && (
+						<Text  style={styles.channelBusyText}>
+							(busy)
+						</Text>
+					)}
```
Comment (phuocnguyenncc) [inline apps/mobile/src/app/screens/home/homedrawer/components/ChannelList/ChannelListItem/index.tsx]: check spacing and add translation

**Review by phuocnguyenncc — APPROVED** [review]

File: libs/translations/src/languages/en/clan.json [hunk 2]
```diff
@@ -8,5 +8,6 @@
   "addClan": "Add Clan",
   "errorMessage": "Please enter a valid clan name (max 64 characters, only words, numbers, _ or -).",
   "duplicateNameMessage" : "The clan name already exists. Please enter another name.",
-  "commutityGuide" : "Community Guidelines."
+  "commutityGuide" : "Community Guidelines.",
+  "busy": "busy"
```
Comment (nguyentrannhan) [inline libs/translations/src/languages/en/clan.json]: Busy

File: libs/translations/src/languages/vi/clan.json [hunk 3]
```diff
@@ -8,6 +8,6 @@
   "addClan": "Tạo clan",
   "errorMessage": "Vui lòng nhập tên ban hội hợp lệ (tối đa 64 ký tự, chỉ từ, số, _ hoặc -).",
   "duplicateNameMessage" : "Tên Clan đã tồn tại. Vui lòng chọn tên khác.",
-  "commutityGuide" : "Quy tắc cộng đồng."
-
+  "commutityGuide" : "Quy tắc cộng đồng.",
+  "busy": "bận rộn"
```
Comment (nguyentrannhan) [inline libs/translations/src/languages/vi/clan.json]: Đang bận

Comment (vyhuynhngoc) [reply libs/translations/src/languages/vi/clan.json]: Em fix xong ạ

**Pull request merged**